### DISCUSSION TITLE
Numero fattura alfa numerico

### DISCUSSION THREAD

**Discussion by antonio8101** [fattureincloud/api#346] [Q&A] [answered]
Avrei la necessità di creare fatture con numerazione alfa numerica: infatti vorremmo numerare le fatture non ad es. 123, ma 123/A. 
L'operazione ``create issue document`` [https://developers.fattureincloud.it/api-reference/#post-/c/-company_id-/issued_documents](https://developers.fattureincloud.it/api-reference/#post-/c/-company_id-/issued_documents), ha mappato il campo ``number`` della fattura come ``numerico``. E' possibile invece inviare testo?

**Comment by emajo** [comment, marked as answer]
Ciao @antonio8101, il campo number è numerico perché contiene il progressivo della fattura, per numerare una fattura 123/A basterà valorizzare il campo `number` a `123`, e il campo `numeration` a `/A`.

Grazie,
Emanuele.
